src/applications/vaos/containers/ReasonForAppointmentPage.jsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import {
+  openFormPage,
+  updateFormData,
+  routeToNextAppointmentPage,
+  routeToPreviousAppointmentPage,
+} from '../actions/newAppointment.js';
+import SchemaForm from 'platform/forms-system/src/js/components/SchemaForm';
+import FormButtons from '../components/FormButtons';
+import { getFormPageInfo } from '../utils/selectors';
+import { PURPOSE_TEXT } from '../utils/constants';
+
+const initialSchema = {
+  type: 'object',
+  required: ['reasonForAppointment'],
+  properties: {
+    reasonForAppointment: {
+      type: 'string',
+      enum: [
+        PURPOSE_TEXT['routine-follow-up'],
```

Comment:
I think you'd just want to use `routine-follow-up` here, you don't need to get the description from the `PURPOSE_TEXT` object. I think you'd want to use that below, in the labels part of `uiSchema`.

Reply:
Where and how is this enum being used when generating the markup?

Reply:
Actually, I'm confused about this change. This was changed based on a previous comment that the ids where out of date.

Reply:
So, generally what we try to do is set the `enum` property to a list of ids. That sets the form to only allow those particular options. These are the values that will get set in our form data when a user chooses a particular option. They're used as the `value` property on the `input` elements in the markup. In addition, we also want to specify a label for each of those ids, which is the text the user sees for each option (the text in the `label` element in the markup).

So, for example in this case, you would make the enum array:

```
enum: [ 'routine-follow-up', 'new-issue', 'medication-concern' ],
```

Or if you want to be slightly fancier: `enum: Object.keys(PURPOSE_TEXT),`

Then, you want to set the `labels` property in `uiSchema` to 

```
labels: PURPOSE_TEXT
```

The form system we're using expects `labels` to be an object with properties for each value in the `enum` array above. Which is what `PURPOSE_TEXT` is, so you don't have to do anything else with it.

There are two ways of specifying labels for the ids in an `enum` array. The `labels` object is one, you can also create an array called `enumNames` beside `enum` in the `schema` object. We tend to use whichever is easier for a particular page.

Reply:
Ok. I made the changes as suggested. Thanks